Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -1,177 +1,233 @@
+            matTooltip="Got Feedback? Than that`s the right place to click."
+            (click)="sidenav.close()"
+          >
+            <mat-icon>info_outline</mat-icon>
+            <span
+              i18n="NavigationComponent|Link to Information page@@NavigationComponentInformation"
+              >Information</span
+            >
+          </button>
+        </mat-list-item>
+        <mat-list-item>
+          <button
+            mat-menu-item
+            feedbackDrawer
+            i18n-matTooltip="navigationBar|Bug Button Tooltip@@navigationBarBugTooltip"
+            matTooltip="Found a Bug? Than that`s the right place to click."
+            (click)="sidenav.close()"
+          >
+            <mat-icon>bug_report</mat-icon>
+            <span
+              i18n="NavigationComponent|Link to Bug page@@NavigationComponentBug"
+              >Report a Bug</span
+            >
+          </button>
+        </mat-list-item>
+      </mat-list>
+    </nav>
+  </mat-sidenav>
+  <mat-sidenav-content>
+    <div class="navigation mat-elevation-z5">
+      <app-online-bar></app-online-bar>
+      <nav>
+        <mat-toolbar color="primary">
+          <mat-toolbar-row class="mat-elevation-z6">
+            <div class="left app-title">
+              <button mat-button [routerLink]="['/']">
+                <span
+                  class="mat-display-1"
+                  id="title"
+                  i18n="NavigationBar|Application Name@@navigationBarApplicationName"
+                  >Phonebook</span
+                >
+                <span
+                  class="beta-tag mat-headline pb-hide-below-small"
+                  *ngIf="environmentService.getEnvironment() == Environment.preview"
+                  i18n="NavigationBar|Tag on Application Name@@navigationBarApplicationNameTag"
+                  >Preview</span
+                >
+                <span
+                  class="beta-tag mat-headline pb-hide-below-small"
+                  *ngIf="environmentService.getEnvironment() == Environment.local"
+                  >Dev</span
+                >
+                <span
+                  class="version pb-hide-below-small"
+                  *ngIf="environmentService.getEnvironment() != Environment.local"
+                >
+                  v{{ version }}</span
+                >
+              </button>
+            </div>
+            <div id="pb-greetings" class="pb-hide-below-big" *ngIf="currentUser != null">
+              {{ (getGreetingMessage() | async) + ' ' + currentUser.Firstname }}!
+            </div>
+            <div class="pb-missing-userimage-info pb-hide-below-med mat-body-2" *ngIf="!hasImage">
+              <button
+                mat-button
+                class="pb-opacity-background"
+                *ngIf="currentUser != null"
+                (click)="navigateToOwnProfile()"
+                i18n="@@MissingUserImage"
```

Comment:
Please add a description and Label as done above. 

Example: `i18n="NavigationBar|Tag on Application Name@@navigationBarApplicationNameTag"`